modules/nf-core/spaceranger/count/main.nf

```diff
@@ -0,0 +1,67 @@
+process SPACERANGER_COUNT {
+    tag "$meta.id"
+    label 'process_high'
+
+    // TODO push to nf-core docker
```

Comment:
is the expectation that this will get set to `nfcore/spaceranger:VERSION` prior to merging?